server/README.md

```diff
@@ -0,0 +1,102 @@
+# preact-render-to-string
```

Comment:
We should probably redo this `preact/server` and maybe something with the logos

Reply:
Good point, I've updated the title and removed the badges. In the longterm we might want to think about removing it or adding readme's for the other addons too. But that's something for later.